src/index.ts

```diff
@@ -484,6 +484,11 @@ async function startServer(
 	port: string | number,
 ): Promise<Server> {
 	let server: Server;
+	// empty 404 error for undefined paths
+	app.use((_request, response) => {
+		response.status(404);
+		response.end();
```

Comment:
NIT: We can chain them for consistency :)
```suggestion
		res.status(404).end();
```